packages/vue/src/composables/form.ts

```diff
@@ -165,7 +171,6 @@ export function useForm<
 					return hooks.error?.(incoming, context)
 				},
 				success: (payload, context) => {
-					clearErrors()
```

Comment:
Is this on purpose? Errors should be cleared on success.

Reply:
They were by reset, but this is now in a different method